tests/FakeItEasy.Analyzer.Tests/NonVirtualSetupAnalyzerTests.cs

```diff
@@ -290,7 +290,7 @@ public class TheClass
         public void Test()
         {
             var foo = A.Fake<IFoo>();
-            A.CallTo(() => foo.Bar);
+            A.CallToSet(() => foo.Bar);

```

Comment:
Unrelated to the PR, but I'm pretty sure it was a mistake. I can put it in a separate PR if you prefer.